elements/rh-spinner/docs/00-overview.md

```diff
@@ -1,6 +1,36 @@
 {% section %}
 ## Overview
 
-Spinner
+A spinner indicates that an action is in progress. It appears as an animated circle over the section that is loading, and it may include a text label.
```

Comment:
move this to jsdoc and use `getElementDescription`

Reply:
I added `@description - A spinner indicates that an action is in progress. It appears as an animated circle over the section that is loading, and it may include a text label` to jsdoc and `{% getElementDescription %}{% endgetElementDescription %}` to the spinner's overview.md, but I'm getting an error with that shortcode. Should it be something else?

Reply:
don't use `@description` - the description field of the jsdoc is everything that *isn't* tagged with a `@jsdocTag`

Reply:
I tried adding it without the @ tag, but it's still saying there's an error with the Nunjucks shortcode.

Reply:
please print the error here